pkg/reconciler.v1/common/service.go

```diff
@@ -42,23 +42,23 @@ var (
 	})
 )
 
-// KubeflowServiceReconciler defines a Service Reconciler for KubeflowJob
-type KubeflowServiceReconciler struct {
+// ServiceReconciler defines a Service Reconciler for KubeflowJob
```

Comment:
Still mentions KubeflowJob here

Reply:
Thanks for pointing out. I changed `KubeflowJob` to `generic training job`.